config/passport.js

```diff
@@ -13,37 +13,19 @@ module.exports = function(passport) {
     });
   });
 
-  passport.use('local-signup', new LocalStrategy({
-    usernameField: 'name',
-    passwordField: 'password',
-    passReqToCallback: true
-  }, function(req, name, password, done) {
-
-    process.nextTick(function() {
-
-      User.findOne({
-        username: name
-      }, function(err, user) {
-        if (err)
-          return done(err);
-
-        if (user) {
-          return done(null, false, req.flash('signupMessage', 'Email Already Exists.'));
-        } else {
-          var newUser = new User();
-          newUser.username = name;
-          newUser.password = newUser.generateHash(password);
-          newUser.save(function(err) {
-            if (err)
-              return err;
-            return done(null, newUser);
-          });
+  passport.use(new LocalStrategy(
```

Comment:
`LocalStrategy` is for username/password authentication. When you create your signin.ejs view, you will POST to this route just like the sign up form. It will check the database for the username and then check if the password is correct. If all works out, we return `done(null, user)`.